src/Resource/ContentType.php

```diff
@@ -82,16 +82,25 @@ public function getDescription()
      * If the field does not exist, null is returned.
      *
      * @param string $fieldId
+     * @param bool   $tryCaseInsensitive
      *
      * @return Field|null
      */
-    public function getField($fieldId)
+    public function getField($fieldId, $tryCaseInsensitive = false)
     {
-        if (!isset($this->fields[$fieldId])) {
-            return null;
+        if (isset($this->fields[$fieldId])) {
+            return $this->fields[$fieldId];
+        }
+
+        if ($tryCaseInsensitive) {
```

Comment:
To be honest I'd rather not, this is a small "hack" that I used to be able to simplify management in the entry class, in order to keep compatibility with some older behavior (which I don't like too much). Because of this, let's keep it "hidden" 😅